pkg/upgrade/job/job.go

```diff
@@ -182,6 +183,9 @@ func NewUpgradeJob(plan *upgradeapiv1.Plan, serviceAccountName, nodeName, contro
 						Env: []corev1.EnvVar{{
 							Name:  "SYSTEM_UPGRADE_PLAN_NAME",
 							Value: plan.Name,
+						}, {
+							Name:  "SYSTEM_UPGRADE_PLAN_LATEST_HASH",
```

Comment:
Why do you inject the hash as env variable into the the pod?

Reply:
It's general purpose and something might need it. 🤷‍♂ 

The latest version is/was there so I decided to replicate that behavior with the hash.